src/utils/db.ts

```diff
@@ -35,11 +35,14 @@ interface UptimeStats {
   uptimeRatio: number;
 }
 
+let uptimeStatisticsCached: Record<string, UptimeStats> | undefined = undefined;
+let lastUptimeRetrieval = 0;
+
```

Comment:
I'm confused why github is showing these as net new lines, as these lines currently exist on the `main` branch --

https://github.com/sarcophagus-org/archaeologist-uptime-monitor/blob/ec94cb00bb3efb16accf67db362842d7b7ded23d/src/utils/db.ts#L38

Reply:
Looks like the only thing that may matter here is that the `onlinePeerIdsCached` check seems to be removed (although it is on `main`)

Tried rebasing locally, but getting `divergent branches` error, and looks like github is calling out merge conflicts as well.